mu/virtual_environment.py

```diff
@@ -822,6 +825,28 @@ def install_jupyter_kernel(self):
                 "Unable to install kernel\n%s" % compact(output)
             )
 
+    def install_from_unpacked_wheels(self, dirpath):
+        #
+        # This command should install the baseline packages, picking up the
+        # precompiled wheels from the wheels path
+        #
+        # For dev purposes (where we might not have the wheels) warn where
+        # the wheels are not already present and download them
+        #

```

Comment:
Should this comment use the conventional docstring format? `"""..."""`

Reply:
Yep; this was a cut-and-paste from a longer function. There'll be a certain amount of smoothing off of this kind to do before the PR is complete.